Gathering entries...
Gathering entries...
Gathering entries...
Gathering entries...
Gathering entries...
Gathering entries...
Gathering entries...
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
    return self.main(*args, **kwargs)
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return self.main(*args, **kwargs)
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
    rv = self.invoke(ctx)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
    rv = self.invoke(ctx)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return __callback(*args, **kwargs)
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    orfs = ProdigalOutput(genome_path)
    return self.main(*args, **kwargs)
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    return ctx.invoke(self.callback, **ctx.params)
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    return ctx.invoke(self.callback, **ctx.params)
    orfs = ProdigalOutput(genome_path)
    for record in SeqIO.parse(self.path, "fasta"):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    for record in SeqIO.parse(self.path, "fasta"):
    rv = self.invoke(ctx)
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    return iterator_generator(handle)
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    return iterator_generator(handle)
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
    orfs = ProdigalOutput(genome_path)
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    return iterator_generator(handle)
    super().__init__(source, mode="t", fmt="Fasta")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    super().__init__(source, mode="t", fmt="Fasta")
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
    super().__init__(source, mode="t", fmt="Fasta")
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    for record in SeqIO.parse(self.path, "fasta"):
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return self.main(*args, **kwargs)
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return self.main(*args, **kwargs)
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    rv = self.invoke(ctx)
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    rv = self.invoke(ctx)
    return ctx.invoke(self.callback, **ctx.params)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    rv = self.invoke(ctx)
    return ctx.invoke(self.callback, **ctx.params)
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
    return ctx.invoke(self.callback, **ctx.params)
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return __callback(*args, **kwargs)
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
    return _process_result(sub_ctx.command.invoke(sub_ctx))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    orfs = ProdigalOutput(genome_path)
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    return __callback(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
    return self.main(*args, **kwargs)
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return self.main(*args, **kwargs)
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    for record in SeqIO.parse(self.path, "fasta"):
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    rv = self.invoke(ctx)
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return ctx.invoke(self.callback, **ctx.params)
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    rv = self.invoke(ctx)
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return iterator_generator(handle)
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    return iterator_generator(handle)
    super().__init__(source, mode="t", fmt="Fasta")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    return _process_result(sub_ctx.command.invoke(sub_ctx))
    super().__init__(source, mode="t", fmt="Fasta")
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return ctx.invoke(self.callback, **ctx.params)
    super().__init__(source, mode="t", fmt="Fasta")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return __callback(*args, **kwargs)
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
    return iterator_generator(handle)
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    self.stream = open(source, "r" + mode)
    return iterator_generator(handle)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return __callback(*args, **kwargs)
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    orfs = ProdigalOutput(genome_path)
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
    super().__init__(source, mode="t", fmt="Fasta")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    for record in SeqIO.parse(self.path, "fasta"):
    orfs = ProdigalOutput(genome_path)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    self.stream = open(source, "r" + mode)
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
Traceback (most recent call last):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/bin/tart-utils", line 33, in <module>
    super().__init__(source, mode="t", fmt="Fasta")
    orfs = ProdigalOutput(genome_path)
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    orfs = ProdigalOutput(genome_path)
    for record in SeqIO.parse(self.path, "fasta"):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    super().__init__(source, mode="t", fmt="Fasta")
    for record in SeqIO.parse(self.path, "fasta"):
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    for record in SeqIO.parse(self.path, "fasta"):
    sys.exit(load_entry_point('bio-tart', 'console_scripts', 'tart-utils')())
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    for record in SeqIO.parse(self.path, "fasta"):
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    super().__init__(source, mode="t", fmt="Fasta")
    return iterator_generator(handle)
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    super().__init__(source, mode="t", fmt="Fasta")
    return __callback(*args, **kwargs)
    super().__init__(source, mode="t", fmt="Fasta")
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    return self.main(*args, **kwargs)
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1078, in main
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    return iterator_generator(handle)
    super().__init__(source, mode="t", fmt="Fasta")
    self.stream = open(source, "r" + mode)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    self.stream = open(source, "r" + mode)
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    self.stream = open(source, "r" + mode)
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    self.stream = open(source, "r" + mode)
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 201, in record_orf_locations
    orfs = ProdigalOutput(genome_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/packages/tart/tart/utils/prodigal.py", line 40, in __init__
    for record in SeqIO.parse(self.path, "fasta"):
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/__init__.py", line 605, in parse
    return iterator_generator(handle)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/FastaIO.py", line 223, in __init__
    super().__init__(source, mode="t", fmt="Fasta")
  File "/users/PDS0325/sachitk26/.conda/envs/tart/lib/python3.11/site-packages/Bio/SeqIO/Interfaces.py", line 45, in __init__
    self.stream = open(source, "r" + mode)
                  ^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/users/PDS0325/sachitk26/packages/tart/validation/orfs/calls/genome_name.faa'
slurmstepd: error: *** JOB 24924279 ON p0203 CANCELLED AT 2023-11-30T13:13:26 ***
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
